jdaviz/configs/imviz/plugins/catalogs/catalogs.py

```diff
@@ -180,14 +207,25 @@ def search(self, error_on_fail=False):
 
         # QTable stores all the filtered sky coordinate points to be marked
         catalog_results = QTable({'coord': filtered_skycoord_table})
+       
         self.number_of_results = len(catalog_results)
+        
 
         # markers are added to the viewer based on the table
         viewer.marker = {'color': 'red', 'alpha': 0.8, 'markersize': 5, 'fill': False}
         viewer.add_markers(table=catalog_results, use_skycoord=True, marker_name=self._marker_name)
 
+        # adding in coordinates and Source IDs into the catalog table
+        for i in range(len(self.app._catalog_source_table)):
+            row_info = {'Right Ascension (degrees)': self.app._catalog_source_table['ra'][i],
+                        'Declination (degrees)': self.app._catalog_source_table['dec'][i],
+                        'Object ID' : self.app._catalog_source_table['objid'][i]
+                       }
+            self.table.add_item(row_info)
```

Comment:
I think you can also access the individual rows, which could remove the need to iterate over an index.  I'm not sure if this will work verbatim, but probably looks something like:

```
for row in self.app._catalog_source_table:
    row_info = {'Right Ascension (degrees)': row['ra'],
                'Declination (degrees)': row['dec'],
                'Object ID' : row['objid']
               }
    self.table.add_item(row_info)